apps/finance/contracts/Finance.sol

```diff
@@ -172,9 +176,9 @@ contract Finance is EtherTokenConstant, IsContract, AragonApp {
         require(_periodDuration >= 1 days, ERROR_INIT_PERIOD_TOO_SHORT);
         settings.periodDuration = _periodDuration;
```

Comment:
For sake of consistency with other `require()` checks, I've opted to keep this check duplicated, since it's a parameter check.

However, I've modified it so it uses a constant instead of a magic number in https://github.com/aragon/aragon-apps/pull/781.